code/Model/SiteTree.php

```diff
@@ -1875,7 +1875,7 @@ public function getCMSFields()
                 'AbsoluteLink' => _t(__CLASS__.'.DependtPageColumnURL', 'URL'),
                 'DependentLinkType' => _t(__CLASS__.'.DependtPageColumnLinkType', 'Link type'),
             );
-            if (class_exists('Subsite')) {
+            if (class_exists(Subsite::class)) {
                 $dependentColumns['Subsite.Title'] = singleton('Subsite')->i18n_singular_name();
```

Comment:
I think you've missed `singleton('Subsite')`

Reply:
This crap itself when Subsite is installed.

Reply:
I obviously didn't test this. Thanks, updated.